test/distributed/elastic/rendezvous/dynamic_rendezvous_test.py

```diff
@@ -1795,14 +1812,27 @@ def test_use_agent_store_is_disabled(self):
 
     @patch.object(dist, "PrefixStore")
     def test_share_tcp_store_from_backend(self, prefix_store_class_mock):
-        prefix_store = Mock(spec=dist.PrefixStore)
-        prefix_store_class_mock.return_value = prefix_store
+        expected_addr = "expected_address"
+        expected_port = 54231
```

Comment:
Shouldn't this be `54321` a.k.a `TEST_PORT` and not `54231`?

Reply:
no, this is indeed expected because we are create a new TCPStore anyway.